cav_msgs/msg/Route.msg

```diff
@@ -25,3 +25,8 @@ int64[] shortest_path_lanelet_ids
 # LaneLet IDs of the whole route, including all lanelets in the shortest path
 int64[] route_path_lanelet_ids
 
+# End point of the route in map frame. Must be on the last lanelet of the route.
+cav_msgs/PathNode end_point
```

Comment:
If you look at the PathNode message spec, you will see it is explicitly defined in an ENU frame. This is not necessarily in the map frame. I would prefer that you use geometry_msgs/Point to represent this data. That way the above comment correctly describes the frame of this data.

Reply:
That makes sense. I changed it to geometry_msgs/Point.